src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/operations/blob.py

```diff
@@ -174,13 +180,13 @@ def upload_append_blob():
             'container_name': container_name,
             'blob_name': blob_name,
             'file_path': file_path,
-            'progress_callback': _update_progress,
+            'progress_callback': get_update_progress_fn(cmd),
```

Comment:
Does this show progress for each file?  If so, it seems very much like with Tamir is trying to do in the image copy extension.  If this works for us, it might work in his situation.

Reply:
yes, it does.

Reply:
Please touch base with him and see if this helps with his image_copy extension, since I'm assuming this doesn't break due to the cli_ctx.